app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -11,8 +11,10 @@ def login_dot_gov
     redirect_to('/login')
   rescue OmniauthError => e
     Rails.logger.error e.message
-    flash[:error] = 'Error logging in. Please reach out to' \
-                    ' search@support.digitalgov.gov if the problem persists'
+    flash[:error] = <<~MESSAGE
+      Error logging in. Please reach out to
+      #{Rails.application.secrets.organization[:support_email_address]} if the problem persists.
```

Comment:
Per our discussion, let's use a constant throughout these changed files, to avoid reading it out of the secrets file each time. We can either:
- move all the email constants out of app/mailers/application_mailer.rb to config/initializers/constants.rb

OR

- just set the email addresses in config/initializers/constants.rb if they are used in a variety of places in the app

Coin flip!